panic: runtime error: invalid memory address or nil pointer dereference [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe3706c]

command used: cat subdomains.txt |katana -jc -d 5 -headless -o katana.txt
katana version: v1.0.5


panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe3706c]

goroutine 49479 [running]:
github.com/projectdiscovery/utils/url.(*URL).String(0x0)
	/home/gdattacker/go/pkg/mod/github.com/projectdiscovery/utils@v0.0.72/url/url.go:98 +0x2c
github.com/projectdiscovery/katana/pkg/engine/hybrid.(*Crawler).navigateRequest.func1(0xc0083e8b40)
	/home/gdattacker/go/pkg/mod/github.com/projectdiscovery/katana@v1.0.5/pkg/engine/hybrid/crawl.go:68 +0x230
github.com/projectdiscovery/katana/pkg/engine/hybrid.(*Hijack).Start.func2(0x0?)
	/home/gdattacker/go/pkg/mod/github.com/projectdiscovery/katana@v1.0.5/pkg/engine/hybrid/hijack.go:52 +0x28
reflect.Value.call({0x13905e0?, 0xc018f737d0?, 0x100c010bcf640?}, {0x1572398, 0x4}, {0xc00188df58, 0x1, 0xc0083e8b40?})
	/usr/local/go/src/reflect/value.go:596 +0xce7
reflect.Value.Call({0x13905e0?, 0xc018f737d0?, 0xc0083e8b40?}, {0xc00188df58?, 0xc00c4b2330?, 0x707260?})
	/usr/local/go/src/reflect/value.go:380 +0xb9
github.com/go-rod/rod.(*Browser).eachEvent.func1()
	/home/gdattacker/go/pkg/mod/github.com/go-rod/rod@v0.114.1/browser.go:401 +0x3d9
github.com/projectdiscovery/katana/pkg/engine/hybrid.(*Hijack).Start.func3()
	/home/gdattacker/go/pkg/mod/github.com/projectdiscovery/katana@v1.0.5/pkg/engine/hybrid/hijack.go:57 +0x22
created by github.com/projectdiscovery/katana/pkg/engine/hybrid.(*Crawler).navigateRequest in goroutine 49419
	/home/gdattacker/go/pkg/mod/github.com/projectdiscovery/katana@v1.0.5/pkg/engine/hybrid/crawl.go:49 +0x51b

--- a/pkg/engine/hybrid/crawl.go
+++ b/pkg/engine/hybrid/crawl.go
@@ -47,7 +47,10 @@ func (c *Crawler) navigateRequest(s *common.CrawlSession, request *navigation.Re
 
 	xhrRequests := []navigation.Request{}
 	go pageRouter.Start(func(e *proto.FetchRequestPaused) error {
-		URL, _ := urlutil.Parse(e.Request.URL)
+		URL, err := urlutil.Parse(e.Request.URL)
+		if err != nil {
+			return errorutil.NewWithTag("hybrid", "could not parse URL").Wrap(err)
+		}
 		body, _ := FetchGetResponseBody(page, e)
 		headers := make(map[string][]string)
 		for _, h := range e.ResponseHeaders {

